src/layout/Footer.tsx

```diff
@@ -15,39 +18,263 @@ export const Footer = () => {
+            <LaunchIcon fontSize="small" />
+          </MuiLink>
+        </Box>
+
+        <Divider sx={{ bgcolor: 'primary.main', mb: '0.8rem' }} />
+
+        <Box
+          sx={{
+            display: 'flex',
+            justifySelf: { xs: 'center', lg: 'start' },
+            gap: '0.5rem',
+            gridColumn: { xs: 1, lg: 3 },
+            mb: '0.3rem',
+          }}>
+          <ArrowRightAltIcon />
+          <MuiLink
+            sx={{
+              color: 'primary.main',
+              display: 'flex',
+              gap: '0.5rem',
+            }}
+            data-testid={dataTestId.footer.newsLink}
+            target="_blank"
+            rel="noopener noreferrer"
+            href={'https://pub.dialogapi.no/s/MjI2NDQ6MWVmZmQxOTktMGVkZi00YjZhLWE2ZjQtNzNmZWU2NjI2ZmFi'}>
+            {t('footer.news_link')}
+            <LaunchIcon fontSize="small" />
+          </MuiLink>
+        </Box>
+
+        <Box
+          sx={{
+            display: 'flex',
+            justifySelf: { xs: 'center', lg: 'start' },
+            gap: '0.5rem',
+            gridColumn: { xs: 1, lg: 3 },
+            mb: '0.3rem',
+          }}>
+          <ArrowRightAltIcon />
+          <MuiLink
+            data-testid={dataTestId.footer.privacyLink}
+            color={'primary.main'}
+            component={Link}
+            to={UrlPathTemplate.PrivacyPolicy}>
+            {t('privacy.privacy_statement')}
+          </MuiLink>
+        </Box>
+
+        <Box
+          sx={{
+            display: 'flex',
+            justifySelf: { xs: 'center', lg: 'start' },
+            gap: '0.5rem',
+            gridColumn: { xs: 1, lg: 3 },
+            mb: '0.3rem',
+          }}>
+          <ArrowRightAltIcon />
+          <MuiLink
+            sx={{
+              color: 'primary.main',
+              display: 'flex',
+              gap: '0.5rem',
+            }}
+            data-testid={dataTestId.footer.availabilityStatement}
+            target="_blank"
+            rel="noopener noreferrer"
+            href={'https://uustatus.no/nb/erklaringer/publisert/bffb4b1d-25eb-4fe0-bac7-2f0b4a8e0fd9'}>
+            {t('about.availability_statement')}
+            <LaunchIcon fontSize="small" />
+          </MuiLink>
+        </Box>
+
+        <Box
+          sx={{
+            display: 'flex',
+            justifySelf: { xs: 'center', lg: 'start' },
+            gap: '0.5rem',
+            gridColumn: { xs: 1, lg: 3 },
+          }}>
+          <ArrowRightAltIcon />
+
+          <Typography
+            paragraph
+            sx={{
+              color: 'primary.main',
+            }}>
+            <Trans t={t} i18nKey="footer.technical_support_link">
+              <MuiLink
+                sx={{
+                  color: 'primary.main',
+                }}
+                href={'mailto:kontakt@sikt.no'}
+                target="_blank"
+                rel="noopener noreferrer">
+                data-testid={dataTestId.footer.technicalSupportLink}
```

Comment:
data-testid står utenfor taggen og brukes dermed ikke nå
```suggestion
          <Typography color="primary">
            <Trans t={t} i18nKey="footer.technical_support_link">
              <MuiLink
                sx={{ color: 'primary.main' }}
                href={'mailto:kontakt@sikt.no'}
                target="_blank"
                rel="noopener noreferrer"
                data-testid={dataTestId.footer.technicalSupportLink}>
```